archetypes/blog-post.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: `<the title of the post>`
+date: <yyyy>-<mm>-<dd>
+slug: <seo-friendly-version-of-title-separated-by-dashes>
+---
+
+**Author:** <your name> (<your organization name>), <another author's name> (<their organization>)
+
+Instructions:
+- Replace these instruction and the following text with your content.
+- Replace `<angle bracket placeholders>` with actual values. For example, you would update `date: <yyyy>-<mm>-<dd>` to look something like `date: 2021-10-21`.
+- For convenience, use third-party tools to author and collaborate on your content.
+- To save time and effort in reviews, check the spelling, grammar, and style of your content before contributing.
+- Feel free to ask for assistance in the Kubernetes Slack channel, [#sig-docs-blog](https://kubernetes.slack.com/archives/CJDHVD54J).
+
+
+Replace this first line of your content with one to three sentences that summarize the blog post.
+
+## This is a section heading
+
+To help reader, organize your content into sections that contain about three to six paragraphs.
+
+To do _x_, enter the following command:
+```bash
+$ This is a command input.
+```
+
+The command output shows you _y_, as shown in the following example.
+```bash
```

Comment:
Don't highlight output as if it's shell.

Reply:
👍

Reply:
Updated. Thanks. Replaced the example with actual content from https://github.com/kubernetes/website/edit/master/content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md